src/vs/workbench/contrib/extensions/electron-browser/extensionTipsService.ts

```diff
@@ -18,7 +18,10 @@ import { ITextModel } from 'vs/editor/common/model';
 import { IStorageService, StorageScope } from 'vs/platform/storage/common/storage';
 import product from 'vs/platform/product/node/product';
 import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
-import { ShowRecommendedExtensionsAction, InstallWorkspaceRecommendedExtensionsAction, InstallRecommendedExtensionAction } from 'vs/workbench/contrib/extensions/browser/extensionsActions';
+// {{SQL CARBON EDIT}}
+import { ShowRecommendedExtensionsAction, InstallWorkspaceRecommendedExtensionsAction, InstallRecommendedExtensionAction, ShowAppLaunchRecommendedExtensionsAction, InstallAppLaunchRecommendedExtensionsAction, } from 'vs/workbench/contrib/extensions/browser/extensionsActions';
+// {{SQL CARBON EDIT}}
```

Comment:
Minor: you can put `// {{SQL CARBON EDIT}} - End` down below the sql file import if you prefer.